src/internal.d.ts

```diff
@@ -39,6 +39,8 @@ export interface Options extends preact.Options {
 	_skipEffects?: boolean;
 	/** Attach a hook that is invoked after an error is caught in a component but before calling lifecycle hooks */
 	_catchError(error: any, vnode: VNode, oldVNode?: VNode | undefined): void;
+	/** Attach a hook that is invoked after a component is unmounted. */
+	_unmount?(vnode: VNode): void;
```

Comment:
Why did we add a type for a hook that isn't used? Afaik we're using the public hook `unmount`

Reply:
oh i didnt know about it, i saw it's used by hooks, and other preact/ modules

Reply:
Should just be `options.unmount` source https://github.com/preactjs/preact/blob/master/src/diff/index.js on line 468

Reply:
oh... got it. i've removed this type.

Reply:
The typing change needs to be reverted too, otherwise the PR looks good to me 👍 Great work!